envoy/README.md

```diff
@@ -157,6 +157,7 @@ For containerized environments, see the [Autodiscovery Integration Templates][11
 | `<INTEGRATION_NAME>` | `envoy`                                     |
 | `<INIT_CONFIG>`      | blank or `{}`                               |
 | `<INSTANCE_CONFIG>`  | `{"openmetrics_endpoint": "http://%%host%%:80/stats/prometheus"}` |
+ **Note**: This is a new default check example. If you previously implemented this integration, see the [legacy example][17].
```

Comment:
nit- can you make the wording more like: https://github.com/DataDog/integrations-core/pull/10839/files#diff-479c19626e9d462720457498c00aa8931a08520647a4d1c2fa0f63e7a4128de5R39

Reply:
```suggestion
 **Note**: The current version of the check (1.26.0+) uses [OpenMetrics][17] for metric collection, which requires Python 3. For hosts that are unable to use Python 3, or if you would like to use a legacy version of this check, refer to the following [config][18].
```